src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/PirateEventUtils.java

```diff
@@ -20,6 +22,11 @@
  */
 public final class PirateEventUtils
 {
+    /**
+     * Folder name for the pirate ship schematics
+     */
+    private static String PIRATESHIP_FOLDER = "/Ships/";
```

Comment:
Please make this lowercase. All minecrafts directories are lowercase for better compatibility with Linux.

Reply:
its uppercase because the schematics folder containing the ship schematics is uppercase aswel, could make it lowercase too. Seems a lot of folders in the schematics are uppercase

Reply:
Yeah... that is annoying then.
Then leave it as is.